src/css/style.css

```diff
@@ -19,6 +19,9 @@
 .d-none {
   display: none !important;
 }
+.m-2 {
+  margin-top: 0.75rem !important;
+}
 .mt-3 {
   margin-top: 1rem !important;
 }
```

Comment:
please change to "mt-" as the `t` hints that is used "-top", if not would make us think it changes margin in all directions

Reply:
@Katsu08 do you think its good if we follow the same sizes as in tailwind?

Reply:
> @Katsu08 do you think its good if we follow the same sizes as in tailwind?

I think it's a good idea because we don't need to create documentation, but just add the classes that we need to use.

https://tailwindcss.com/docs/margin

Reply:
Let's do this in another issue!